molo/commenting/wagtail_hooks.py

```diff
@@ -0,0 +1,122 @@
+from daterange_filter.filter import DateRangeFilter
+from django.conf.urls import url
+from django.core.urlresolvers import reverse
+from django.http import HttpResponse
+from import_export import resources
+from molo.commenting.admin import MoloCommentAdmin, CannedResponseModelAdmin
+from molo.commenting.models import MoloComment, CannedResponse
+from molo.commenting.views import WagtailCommentReplyView
+from wagtail.wagtailcore import hooks
+from wagtailmodeladmin.options import ModelAdmin, wagtailmodeladmin_register, \
+    ModelAdminGroup
+from wagtailmodeladmin.views import IndexView
+
+
+@hooks.register('register_admin_urls')
+def register_admin_reply_url():
+    return [
+        url(r'comment/(?P<parent>\d+)/reply/$',
+            WagtailCommentReplyView.as_view(),
+            name='wagtail-molo-comment-reply'),
+    ]
+
+
+class DateFilter(DateRangeFilter):
+    template = 'admin/comments_date_range_filter.html'
+
+
+class CommentsResource(resources.ModelResource):
+    class Meta:
+        model = MoloComment
+        exclude = ('id', 'comment_ptr', 'content_type', 'object_pk',
+                   'site', 'user', 'user_url', 'lft', 'rght',
+                   'tree_id', 'level', 'ip_address', )
+
+
+class ModelAdminTemplate(IndexView):
```

Comment:
Please rename to CommentsAdminView, rename the associated template and move to admin_views.py